README.md

```diff
@@ -73,7 +73,7 @@ To get a local copy up and running, follow these steps:
 
 
   git clone https://github.com/Fahim-Khan-P/LeaderBoard.git
-  
+
```

Comment:
- [ ] OPTIONAL
Kindly ensure to add the _Future Features_ section to your ReadMe, which should include 1-3 potential features to be added.